<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainMenu</class>
 <widget class="QMainWindow" name="MainMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Morse Academy</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(102, 106, 92);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="menuStack">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>0</y>
      <width>671</width>
      <height>601</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(204, 193, 177);</string>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="topMenu">
     <widget class="QLabel" name="gameTitleLabel">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>60</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Welcome to Morse Academy!</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="studyMenu">
     <property name="font">
      <font>
       <family>MS Reference Sans Serif</family>
      </font>
     </property>
     <widget class="QLabel" name="studyLabel">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>10</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Study</string>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>80</y>
        <width>160</width>
        <height>241</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="levelButtonsLayout">
       <item>
        <widget class="QPushButton" name="level1Button">
         <property name="text">
          <string>Level 1: A - E</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="level2Button">
         <property name="text">
          <string>Level 2: F - J</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="level3Button">
         <property name="text">
          <string>Level 3: K - P</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="level4Button">
         <property name="text">
          <string>Level 4: Q - U</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="level5Button">
         <property name="text">
          <string>Level 5: V - Z</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="level6Button">
         <property name="text">
          <string>Level 6: Mastery</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="level7Button">
         <property name="text">
          <string>Level 7: Random</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="studyLevel">
     <widget class="QLabel" name="studyLevelLabel">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>10</y>
        <width>241</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Study Level: </string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QTextEdit" name="userInput">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>500</y>
        <width>501</width>
        <height>70</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="studyMorseLabel">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>150</y>
        <width>401</width>
        <height>241</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="lineWidth">
       <number>5</number>
      </property>
      <property name="text">
       <string>area for morse code level study here</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="nextQuestionButton">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>500</y>
        <width>111</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Next Question</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="translateMenu">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>249</x>
        <y>250</y>
        <width>191</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>translate page </string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>111</width>
      <height>158</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="menuButtonsLayout" stretch="0,0,0">
     <property name="spacing">
      <number>20</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="menuButton">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(204, 193, 177);
</string>
       </property>
       <property name="text">
        <string>Main Menu</string>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="studyButton">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(204, 193, 177);</string>
       </property>
       <property name="text">
        <string>Study!</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="translateButton">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(204, 193, 177);</string>
       </property>
       <property name="text">
        <string>Translate</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
